docs/source/usage.rst

```diff
@@ -37,8 +37,10 @@ As for e.g. :py:class:`sklearn.decomposition.PCA` we first need to
     msPCA.fit(X)
     X_transformed = msPCA.transform(X)
 
+
 Alternatively, this can be achieved in one step by
-:py:meth:`multispaeti.MultispatiPCA.fit_transform`
+:py:meth:`multispaeti.MultispatiPCA.fit_transform` which is also more performant and
+avoids redundant computation
```

Comment:
there is no need to write avoids redundant computation, if you are writing more performant